ksqldb-serde/src/main/java/io/confluent/ksql/serde/json/JsonSchemaProperties.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.serde.json;
+
+import com.google.common.collect.ImmutableSet;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.Map;
+
+public class JsonSchemaProperties extends JsonProperties {
```

Comment:
Should we directly `extend ConnectProperties` ? Not sure if having `JsonProperties` at all make sense?

Reply:
See comment above, I think extends `ConnectProperties` is fine.